Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -2634,35 +2645,26 @@ namespace AssetProcessor
             // examination occurs here.
             // first, is it a source or is it a product in the cache folder?
             QString normalizedPath = examineFile.m_fileName;
-
-            AZ_TracePrintf(AssetProcessor::DebugChannel, "ProcessFilesToExamineQueue: %s delete: %s.\n", examineFile.m_fileName.toUtf8().constData(), examineFile.m_isDelete ? "true" : "false");
-
-            // debug-only check to make sure our assumption about normalization is correct.
-            Q_ASSERT(normalizedPath == AssetUtilities::NormalizeFilePath(normalizedPath));
```

Comment:
Authors note: Github can't detect it but these lines were just moved downwards, after the fixup.